crates/bevy_ecs/src/system/commands.rs

```diff
@@ -281,22 +361,26 @@ impl Commands {
         self
     }
 
+    /// Adds a command directly to the command list. Prefer this to `add_command_boxed` if the type of |command| is statically known.
```

Comment:
is `|command|` doc syntax im not aware of or should this be `` `command` ``

Reply:
It should be `` `command` ``... `|command|` is syntax from the chromium project :grin: